lib/json.bash

```diff
@@ -75,7 +75,7 @@ JSON.keys() {
   JSON.object "$@" |
     cut -f1 |
     sed "s/^\///; s/\/.*//" |
-    sort -u
+    uniq
```

Comment:
@dolik-rce good point. There might be situations (depending on how `put` would be implemented, for example) where they keys are not on adjacent lines.
I guess that why I'd put in the sort in the first place ;-)